packages/lisk-p2p/src/p2p_types.ts

```diff
@@ -44,28 +44,45 @@ export interface P2PConfig {
 	readonly connectTimeout: number;
 	readonly ipAddress?: string;
 	readonly seedPeers: ReadonlyArray<PeerInfo>;
+	readonly version: string;
 	readonly wsEngine?: string;
 	readonly wsPort: number;
-	readonly version: string;
 }
 
-export interface NetworkStatus {}
+// Network info exposed by the P2P library.
+export interface P2PNetworkStatus {
```

Comment:
unConnectedPeers is different because some peers in the _newPeers map might be connected. 

I think newPeers is OK. Once we start moving peers into the _triedPeers table, it will make more sense. So there will be newPeers, triedPeers and connectedPeers. I think we should use the same name to refer to the same thing.

Reply:
Actually, according to the definition in LIPS, the moment you make a connection with any peer from `newPeer` we remove it from there and add it to `triedPeer`. So, `newPeer` should never have any peer for which we made connection already.  Whereas `triedPeers` will have peers with whom we have a connection or had a connection in the past.

so, active peers should have, triedPeers peers which have a connection already and available peers will be the combination of (newPeers + triedPeers(with no socket))

Reply:
We could have `availablePeers` be `newPeers + triedPeers` and `connectedPeers` be peers from the `PeerPool`?

Reply:
> availablePeers be newPeers + triedPeers and connectedPeers be peers from the PeerPool

I think this is good, im not sure if we need to expose `newPeers` or `triedPeers`

Reply:
Isn't `availablePeers` and `connectedPeers` is enough? I thought we don't want to expose newPeers and triedPeers